Doc/library/bdb.rst

```diff
@@ -352,20 +359,20 @@ Finally, the module defines the following functions:
 
 .. function:: checkfuncname(b, frame)
 
-   Check whether we should break here, depending on the way the breakpoint *b*
-   was set.
+   Return True if we should break here, depending on the way the breakpoint
+   *b* was set.
 
-   If it was set via line number, it checks if ``b.line`` is the same as the one
-   in the frame also passed as argument.  If the breakpoint was set via function
-   name, we have to check we are in the right frame (the right function) and if
-   we are in its first executable line.
+   If it was set via line number, it checks if ``b.line`` is the same as the
+   one in *frame*.  If the breakpoint was set via function name, we have to
+   check we are in the right *frame* (the right function) and if we are on
+   its first executable line.
 
 .. function:: effective(file, line, frame)
 
    Determine if there is an effective (active) breakpoint at this line of code.
-   Return a tuple of the breakpoint and a boolean that indicates if it is ok
-   to delete a temporary breakpoint.  Return ``(None, None)`` if there is no
-   matching breakpoint.
+   Return a tuple of the breakpoint that was triggered and a boolean that
+   indicates if it is ok to delete a temporary breakpoint.  Return
+   ``(None, None)`` if there is no matching breakpoint.
```

Comment:
I've made these changes.  I also made other changes to add all the attributes for Breakpoint and then use those as links in this description of `effective`.  Sorry to add so much to the second review, but I think it helps a lot in understanding the breakpoints.  One thing that I didn't say is that `ignore` would be set by the caller.  Not sure if I need to mention that it's a read/write attribute?